app/Models/UserWidget.php

```diff
@@ -15,25 +15,16 @@ class UserWidget extends Model
 
     // ---- Define Relationships ----
 
-    /**
-     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
-     */
     public function user()
     {
         return $this->belongsTo('App\Models\User', 'user_id');
     }
 
-    /**
-     * @return \Illuminate\Database\Eloquent\Relations\HasOne
-     */
     public function widget()
     {
-        return $this->hasOne('App\Models\Widgets', 'widget_id');
+        return $this->hasOne('App\Models\Widget', 'widget_id');
```

Comment:
`App\Models\Widget` doesn't exist?

Reply:
Added, although this is one of those things I don't think should be stored in the DB...